schemas/rum/view-schema.json

```diff
@@ -360,6 +360,39 @@
             }
           },
           "readOnly": true
+        },
+        "display": {
+          "type": "object",
+          "description": "Display properties",
+          "properties": {
+            "scroll": {
+              "type": "object",
+              "description": "Scroll properties",
+              "properties": {
+                "max_depth": {
+                  "type": "number",
+                  "description": "Maximum scrolled distance on this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_scroll_height": {
+                  "type": "number",
+                  "description": "Maximum scrollable distance on this view (in pixels)",
+                  "readOnly": true
+                },
+                "max_depth_time": {
+                  "type": "number",
+                  "description": "Timestamp of the scroll event that reached the maximum scroll depth for this view",
+                  "readOnly": true
+                },
+                "max_scroll_top": {
+                  "type": "number",
+                  "description": "Maximum amount of pixels scrolled",
+                  "readOnly": true
```

Comment:
#
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should these properties be required?